### DISCUSSION TITLE
Translation Platform now live!

### DISCUSSION THREAD

**Discussion by Spiritreader** [AutoDarkMode/Windows-Auto-Night-Mode#614] [Announcements]
Dear Contributors,

we have added crowdin to ease the workload on our awesome translators!
You can find our project here: https://hosted.weblate.org/projects/auto-dark-mode/app

Thanks again to all of our translators, we hope this will improve the experience you have in localizing ADM!
If you are a resident translator and would like to partake in proof-reading please let me know.

Update: We haved moved to weblate.
Crowdin will persist for a while until it will be deprecated in favor of weblate.

**Comment by comradekingu** [comment]
Happy to help if you use Weblate instead. Read the Crowdin terms of service and corresponding documents.

**Reply by Spiritreader** [reply]
I'm not exactly sure what's wrong with crowdin, but I also wouldn't mind moving to weblate as from what I've seen the github integration is better.

However, it seems like weblate has no free tier and we're very, very far away from being able to afford spending money regularily on ADM.
So unless there's some sort of "free for open source" tier I don't think it'll be possible and we have to refer those who don't want to use crowdin to manually editing the resx files instead.

**Reply by comradekingu** [reply]
There is :)
https://weblate.org/en/hosting/
>Weblate proudly supports libre projects
If you feel like being supported by Weblate will help you, [set up your libre project](https://hosted.weblate.org/hosting/) and get the Libre plan gratis.

Crowdin is closed source, has horrendous terms, and doesn't allow spanning large amounts of strings or multiple languages at once. The voting system cements mistakes to where nobody can fix them without getting yet another vote. This in turn means there is no consistency anywhere.

**Reply by Spiritreader** [reply]
Doesn't self hosted mean that I need to provide my own server to host the instance?

I'll try out the Libre project and will update once I know more.

Thanks!

**Reply by comradekingu** [reply]
Yes, but that is only an option.
_Hosted_ is an instance of Weblate,of which there are many.
The Libre plan on Hosted is a no-cost option for libre software projects.

**Reply by Spiritreader** [reply]
I have sent a libre plan approval request and will switch our translation platform to weblate if it gets accepted.

**Reply by Spiritreader** [reply]
We have been approved. Our translation platform has now been moded to weblate.

I've also added the language file for Norwegian Bokmål,
